go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/gballet/go-verkle
 
 go 1.19
 
-require github.com/crate-crypto/go-ipa v0.0.0-20230601170251-1830d0757c80
+require github.com/crate-crypto/go-ipa v0.0.0-20230622122819-a63eaa2f7c20
```

Comment:
Using tenative https://github.com/crate-crypto/go-ipa/pull/51
I'll change when it gets merged.